make/test/JtregNativeHotspot.gmk

```diff
@@ -851,6 +851,7 @@ ifeq ($(call isTargetOs, linux), true)
   BUILD_TEST_exeinvoke_exeinvoke.c_OPTIMIZATION := NONE
   BUILD_HOTSPOT_JTREG_EXECUTABLES_LIBS_exeFPRegs := -ldl
   BUILD_HOTSPOT_JTREG_LIBRARIES_LIBS_libAsyncGetCallTraceTest := -ldl
+  BUILD_HOTSPOT_JTREG_LIBRARIES_LDFLAGS_libfast-math := -ffast-math
```

Comment:
Is the flag redundant by now? The test explicitly works with corresponding platform-specific registers.

Reply:
it's belt-and braces (or belt-and-suspenders in American). This way, we can still do something useful for platforms not fully supported by mainline. I don't want to add ifdefs for every platform to the test.